src/models/quote-db.model.ts

```diff
@@ -0,0 +1,40 @@
+export interface BaseQuoteDbModel {
+    agreementTimestamp: number;
+    gasFeeOnWei: bigint;
+    nonce: bigint;
+    penaltyFeeOnWei: bigint;
+    btcRefundAddress: string;
+    lbcAddress: string;
+    lpBtcAddress: string;
+    rskRefundAddress: string;
+    liquidityProviderRskAddress: string;
+  }
+  
+  export interface PeginQuoteDbModel extends BaseQuoteDbModel {
+    callFeeOnSatoshi: bigint;
+    callOnRegister: boolean;
+    confirmations: number;
+    contractAddr: string;
+    data: string;
+    fedBTCAddr: string;
+    gasLimit: number;
```

Comment:
should `gasLimit` be bigint?

Reply:
we are expecting a number
https://github.com/rsksmart/unified-bridges-sdk/blob/Stable-Test/packages/flyover-sdk/src/api/bindings/data-contracts.ts#L243
should we perform some kind of conversion anyways?

Reply:
would be good to change to bigint in that place too.